scripts/build_docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+DOCS_ROOT=dist/docs/
+REPO_NAME="ooni/probe-cli"
+COMMIT_HASH=$(git rev-parse --short HEAD)
+
+mkdir -p $DOCS_ROOT
+
+strip_title() {
+    # Since the title is already present in the frontmatter, we need to remove
+    # it to avoid duplicate titles
+    local infile="$1"
+    cat $infile | awk 'BEGIN{p=1} /^#/{if(p){p=0; next}} {print}'
+}
+
+cat <<EOF>$DOCS_ROOT/00-index.md
```

Comment:
You should in here actually make an inventory of all the docs that exist in this repo and copy them over to the correct `probe` directory so we can render them.

Be sure to setup a reasonable slug for each of them so that the permalink is correct.